### DISCUSSION TITLE
How to make Mojo 🔥 work on Android and iOS. Watch  👇

### DISCUSSION THREAD

**Discussion by pharaohAi** [modular/modular#1971] [Q&A]
I wonder if it is possible to create Android and iOS applications using the most advanced artificial intelligence programming language in the world, Mojo 🔥 Finally, is it possible to make Max also work on Android and iOS?

These questions never leave my mind, and I do not know who will answer these questions for me, my friend. If you know the answer to these questions, Do not hesitate to answer us

**Comment by hogepodge** [comment]
Thanks for your interest. Android and iOS are not supported at this time. You can find the MAX roadmap here. https://docs.modular.com/max/roadmap
